paris-processor/src/main/java/com/airbnb/paris/processor/framework/SkyProcessor.kt

```diff
@@ -11,20 +11,24 @@ abstract class SkyProcessor : AbstractProcessor() {
         lateinit var INSTANCE: SkyProcessor
     }
 
-    lateinit var filer: Filer
-    lateinit var messager: Messager
-    lateinit var elements: Elements
-    lateinit var types: Types
+    val filer: Filer by lazy { processingEnv.filer }
```

Comment:
changed these to be vals by using lazy. seems better than lateinit vars

Reply:
👍 I hadn't realized the processing environment was always available. Makes sense though.

Reply:
I didn't notice at first either, turns out it's just protected